base/bucket_gocb.go

```diff
@@ -228,29 +229,29 @@ func (bucket *CouchbaseBucketGoCB) retrievePurgeInterval(uri string) (int, error
 
 	resp, err := bucket.mgmtRequest(http.MethodGet, uri, "application/json", nil)
 	if err != nil {
-		return 0, err
+		return time.Duration(0), err
```

Comment:
Everywhere we're doing `time.Duration(0)` seems unnessesary, as the variables are already typed, and `0` is a valid literal for a duration. Can put these back to reduce the changes in the PR.

Reply:
I thought it would be more readable, but I agree with your suggestion, this can be removed.